--- flann-1.9.2/CMakeLists.txt.orig	2021-04-12 09:15:28.000000000 +0200
+++ flann-1.9.2/CMakeLists.txt	2023-01-17 18:18:25.687565563 +0100
@@ -164,7 +164,7 @@ else()
 endif()
 
 # install and export variables
-set(config_install_dir "lib/cmake/${PROJECT_NAME}")
+set(config_install_dir "lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}")
 set(generated_dir "${CMAKE_CURRENT_BINARY_DIR}/generated")
 set(version_config "${generated_dir}/flann-config-version.cmake")
 set(project_config "${generated_dir}/flann-config.cmake")
